components/prombench/manifests/results/prometheus-meta.yaml

```diff
@@ -112,8 +121,7 @@ spec:
         args:
         - "--config.file=/etc/prometheus/config/prometheus.yaml"
         - "--storage.tsdb.path=/data"
-        - "--storage.tsdb.min-block-duration=15m"
-        - "--storage.tsdb.max-block-duration=4h"
```

Comment:
this should have been in a separate PR
but in general these are added to triger compaction more often.
Withou our long running tests don't think this is necessary.